tests/behat/features/capabilities/follow-users/follow-users.feature

```diff
@@ -0,0 +1,96 @@
+@api @follow-users @stability @YANG-7603 @stability-4
```

Comment:
@volodymyr-sydor  When run test second time we will receive this error:

`Drupal\Core\Config\PreExistingConfigException: Configuration objects (social_follow_user.settings) provided by social_follow_user already exist in active configuration in /var/www/html/core/lib/Drupal/Core/Config/PreExistingConfigException.php:65`

![dd0c13050b4a32ccec40e7ad6e2e6d12](https://user-images.githubusercontent.com/10220937/176674765-c5705ba0-0084-4fbd-9534-b1fe3221d20b.png)

It because of config was not remove when module was uninstalled. Can we add those fixes in this PR?
cc @ribel

Reply:
I will check it but in `social_follow_user_uninstall` we already have:
  // Cleanup settings.
  `\Drupal::configFactory()->getEditable('social_follow_user.settings')->delete();`

BTW: Similar things could happen because of failing tests. Could you make sure that that first run was successful, please

Reply:
Thank you, I will recheck it

Reply:
Fixed